src/components/pages/order/navbar/NavbarRightSide.js

```diff
@@ -1,11 +1,41 @@
-import { Link } from "react-router-dom";
+
 import styled from "styled-components";
+import ToggleButton from "../../../reusable-ui/ToggleButton";
 import Profile from "./Profile";
+import { useState } from "react"
+import ToastAdmin from "./ToastAdmin";
+import { toast } from "react-toastify";
+
 
 export default function NavbarRigthSide({username}) {
+
+  const [isModeAdmin, setIsModeAdmin] = useState(false);
```

Comment:
isAdmin, setIsAdmin devrait suffire je pense, on comprends de suite 

"isAdmin" > est-on en admin ?
"isModeAdmin" > est-on en mode admin ? 

cela veux dire la même chose mais le premier et plus court et concis à mon sens

```suggestion
  const [isAdmin, setIsAdmin] = useState(false);
```

Reply:
En principe, je suis d'accord avec toi @Ovgodd 💯 
Après, si @inesmokaddem préfère isModeAdmin, ça, à la liiimite, ça peut passer car c'est pas hyper long et si c'est plus clair pour elle. Là c'est les gouts et les couleurs. Mais si c'était hyper long, oui, faut essayer de raccourcir autant que possible **sans perdre le sens**.